db/migrate/20190215145600_create_pages.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreatePages < ActiveRecord::Migration[5.2]
+  def change
+    create_table :pages do |t|
+      t.string :title
+      t.text :description
+      t.string :layout
+      t.references :group, foreign_key: true
+      t.timestamps
+    end
+    change_table :groups do |t|
```

Comment:
This would mean that a group can only have one page, I don't think that's the expectation. 

We need a join table.

Reply:
Groups should be able to have multiple pages.

Reply:
So yeah, what I have is  just the ability to add multiple pages to a group, and multiple groups to a page, but the association is not there.

Reply:
Updated page to have belongs_to group, and group still has_many pages.

Reply:
Fixed associations using new join table group_page. Has many to has many on groups and pages.